src/views/forms/AccountManagementForms/WithdrawForm/WithdrawButtonAndReceipt.tsx

```diff
@@ -140,16 +138,16 @@ export const WithdrawButtonAndReceipt = ({
       key: 'estimated-route-duration',
       label: <span>{stringGetter({ key: STRING_KEYS.ESTIMATED_TIME })}</span>,
       value:
-        typeof summary?.estimatedRouteDuration === 'number' ? (
+        summary != null && typeof summary?.estimatedRouteDuration === 'number' ? (
```

Comment:
do we need optional chaining on `summar.estimatedRouteDuration` ? since we know that summary is not a nullish value